polaris/ocean/config/single_layer.yaml

```diff
@@ -0,0 +1,19 @@
+omega:
+  time_integration:
+    config_time_integrator: RK4
+  cvmix:
+    config_use_cvmix: false
+  ALE_vertical_grid:
+    config_vert_coord_movement: impermeable_interfaces
+    config_ALE_thickness_proportionality: weights_only
+  debug:
+    config_disable_thick_vadv: true
+    config_disable_thick_sflux: true
```

Comment:
```suggestion
```
I don't think surface fluxes should be off. I needed this to test the river runoff fluxes in the drying_slope case and it could be used for other coastal runs.